### PR TITLE
Add benchmarking results

### PR DESCRIPTION
**Pull request by Rocamonde**
## Description

Add benchmarking results and hyperparameters, previous to publishing the paper preprint.

### CODE REVIEW

**Comment by Rocamonde** [comment]
@taufeeque9 please take a look if you can!

**Review by taufeeque9 — APPROVED** [review]
Looks good. Thanks for writing the util file and adding all the hyperparameter configs!

We can update the files with the argument for the demonstration data once #629 gets approved.

File: benchmarking/util.py
Comment (AdamGleave) [inline benchmarking/util.py:92]: Style guide violation: https://google.github.io/styleguide/pyguide.html#313-imports-formatting Imports should always be at top level of module to make it clear what modules file depends on, unless there's an explicit need for them to be conditional/only locally imported.

I'd also suggest moving all this to a `main()` function that's called in the `__main__` guard block to avoid polluting the global namespace.

**Review by AdamGleave — COMMENTED** [review]
One small linting issue that I fixed for you. Please do try and make tests green before requesting review ;)

Some minor suggested changes on `util.py`, after that I think it should be good to merge.

**Comment by Rocamonde** [comment]
Thanks! Just added those changes. Should be good to merge then once tests pass.

**Comment by AdamGleave** [comment]
Tests are failing on Windows now? Seems like an unrelated error so perhaps best fixed in another PR.

**Comment by Rocamonde** [comment]
Everything is green now but codecov is not sending the results over. Coverage should be unchanged anyways (as can be checked on https://app.codecov.io/gh/HumanCompatibleAI/imitation/pull/627) @AdamGleave can you re-trigger codecov sending the info / override?

**Review by AdamGleave — APPROVED** [review]
LGTM

**Pull request merged**
